src/Tween.ts

```diff
@@ -386,11 +405,7 @@ export class Tween<T extends UnknownProps> {
 					this._reversed = !this._reversed
 				}
 
-				if (this._repeatDelayTime !== undefined) {
-					this._startTime = time + this._repeatDelayTime
-				} else {
-					this._startTime = time + this._delayTime
-				}
+				this._startTime += durationAndDelay * completeCount
```

Comment:
@MasatoMakino Interesting. Looks like this fixes an inaccuracy! Before we had `_startTime = time + this._repeatDelayTime`, which means that if the `time` is beyond the beginning of a repeat, then the next repeat will happen after `this._repeatDelayTime` plus a small value, but it should instead happen at exactly the right time so that the _next_ time is relative to the correct starting time.

Difficult to describe with words. Good find!

Reply:
I appreciate the review!